phpunit.xml.dist

```diff
@@ -5,35 +5,12 @@
 	convertErrorsToExceptions="true"
 	convertNoticesToExceptions="true"
 	convertWarningsToExceptions="true"
+	defaultTestSuite="default"
 	>
 	<testsuites>
-		<testsuite name="performance-lab">
+		<testsuite name="default">
 			<directory suffix=".php">./tests</directory>
 			<exclude>./tests/utils</exclude>
-			<exclude>./tests/plugins</exclude>
-		</testsuite>
-		<testsuite name="auto-sizes">
-			<directory suffix=".php">./tests/plugins/auto-sizes</directory>
-		</testsuite>
-		<testsuite name="dominant-color-images">
-			<directory suffix=".php">./tests/plugins/dominant-color-images</directory>
-		</testsuite>
-		<testsuite name="embed-optimizer">
-			<directory suffix=".php">./tests/plugins/embed-optimizer</directory>
-		</testsuite>
-		<testsuite name="optimization-detective">
-			<directory suffix=".php">./tests/plugins/optimization-detective</directory>
-		</testsuite>
-		<testsuite name="speculation-rules">
-			<directory suffix=".php">./tests/plugins/speculation-rules</directory>
-		</testsuite>
-		<testsuite name="webp-uploads">
-			<directory suffix=".php">./tests/plugins/webp-uploads</directory>
 		</testsuite>
 	</testsuites>
-	<groups>
-		<exclude>
-			<group>ms-required</group>
-		</exclude>
-	</groups>
```

Comment:
Neither the PL plugin nor standalone plugin uses this group at the moment. Once this is required, it can be added in required plugin config.